bc_obps/reporting/models/rls_configs/activity_json_schema.py

```diff
@@ -0,0 +1,14 @@
+from reporting.enums.enums import ReportingTableNames
+from rls.enums import RlsRoles, RlsOperations
+from rls.utils.helpers import generate_rls_grants
+
+
+class Rls:
+    role_grants_mapping = {
+        RlsRoles.INDUSTRY_USER: [RlsOperations.SELECT],
+        RlsRoles.CAS_DIRECTOR: [RlsOperations.SELECT],
+        RlsRoles.CAS_ADMIN: [RlsOperations.SELECT],
+        RlsRoles.CAS_ANALYST: [RlsOperations.SELECT],
+        RlsRoles.CAS_VIEW_ONLY: [RlsOperations.SELECT],
```

Comment:
As another note for some potential refactor: we could introduce a `RlsRoles.ALL` "meta-role" to shorten this and avoid refactoring if we needed to add a role at a later point